### PR TITLE
Variable exporting fix on AWS Terraform Guide

### PR DESCRIPTION
**Pull request by corkrean**
There was an issue with the formatting when exporting the bastion and auth IP. 

The name-value pairs must be enclosed in their own set of quotes, they should not be lumped together under a single set of quotes. 
There must be a space between the two pairs of name/value pairs. 

(I ensured I was running the latest version of AWS CLI 2 and the latest version of Terraform)

### CODE REVIEW

**Comment by inertial-frame** [comment]
To summarize: this updates the AWS [describe-instances](https://docs.aws.amazon.com/cli/latest/reference/ec2/describe-instances.html) example from:

* `aws ec2 describe-instances --filters "Name=name1,Values=val1,Name=name2,Values=val2"`
* to `aws ec2 describe-instances --filters "Name=name1,Values=val1" "Name=name2,Values=val2"`

Indeed, multiple values should be white-space delimited not by a comma:

```
aws ec2 describe-instances \
    --filters Name=instance-type,Values=t2.micro,t3.micro Name=availability-zone,Values=us-east-2c
```

Great catch! LGTM!

**Review by inertial-frame — APPROVED** [review]

**Review by webvictim — APPROVED** [review]

File: docs/pages/aws-terraform-guide.mdx
Comment (r0mant) [inline docs/pages/aws-terraform-guide.mdx:428]: Hmm this is a weird syntax (how does the command determine that the second filter is a --filters flag value and not a positional argument?) but it seems to be indeed correct according to https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Filtering.html examples. @corkrean Have you verified it works?

**Review by r0mant — APPROVED** [review]

File: docs/pages/aws-terraform-guide.mdx
Comment (corkrean) [reply docs/pages/aws-terraform-guide.mdx:428]: Yes, I have verified it.

**Review by russjones — APPROVED** [review]
Bot.

**Comment by webvictim** [comment]
Hi @corkrean,

Thanks for your contribution. We are currently unable to run our tests on external contributions. As such, I'll be helping you validate and merge your pull request. I've created an internal PR [here](https://github.com/gravitational/teleport/pull/6973) and the CI came up clean as seen here: https://drone.gravitational.io/gravitational/teleport/11917

Please wait for another approver and I'll get your code merged.

**Comment by webvictim** [comment]
https://github.com/gravitational/teleport/pull/6973 is merged, closing this. Thanks!

**Pull request closed without merging**
